python/_tskitmodule.c

```diff
@@ -865,14 +890,17 @@ IndividualTable_init(IndividualTable *self, PyObject *args, PyObject *kwds)
 }
 
 static PyObject *
-IndividualTable_add_row(IndividualTable *self, PyObject *args, PyObject *kwds)
+IndividualTable_add_or_update_row(IndividualTable *self, PyObject *args, PyObject *kwds)
```

Comment:
I don't like this pattern - trying to back-infer how a method was called always seems like a bad idea, and leads to convoluted logic.